packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -4,20 +4,26 @@ import {
   MsgDelegate,
   MsgDelegateResponse,
 } from '@agoric/cosmic-proto/cosmos/staking/v1beta1/tx.js';
+import {
+  QueryBalanceRequest,
+  QueryBalanceResponse,
+} from '@agoric/cosmic-proto/cosmos/bank/v1beta1/query.js';
+import { Any } from '@agoric/cosmic-proto/google/protobuf/any.js';
+import { RequestQuery } from '@agoric/cosmic-proto/tendermint/abci/types.js';
 import { AmountShape } from '@agoric/ertp';
 import { makeTracer } from '@agoric/internal';
 import { UnguardedHelperI } from '@agoric/internal/src/typeGuards.js';
 import { M, prepareExoClassKit } from '@agoric/vat-data';
 import { TopicsRecordShape } from '@agoric/zoe/src/contractSupport/index.js';
 import { decodeBase64 } from '@endo/base64';
 import { E } from '@endo/far';
-import { Any } from '@agoric/cosmic-proto/google/protobuf/any.js';
+import { ChainAddressShape } from '../typeGuards.js';
 
 /**
- * @import { ChainAccount, ChainAddress } from '../types.js';
+ * @import { ChainAccount, ChainAddress, CosmosValidatorAddress, ICQConnection } from '../types.js';
  * @import { RecorderKit, MakeRecorderKit } from '@agoric/zoe/src/contractSupport/recorder.js';
  * @import { Baggage } from '@agoric/swingset-liveslots';
- * @import {AnyJson} from '@agoric/cosmic-proto';
+ * @import { AnyJson, RequestQueryJson } from '@agoric/cosmic-proto';
```

Comment:
no padding for types lists, only for object properties
```suggestion
 * @import {AnyJson, RequestQueryJson} from '@agoric/cosmic-proto';
```

Reply:
Acknowledged, updated one of the files I touched during other feedback and will prefer this going forward